lib/internal/modules/typescript/typescript.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
```

Comment:
Each new file added to `lib/internal/modules` slows down startup time a little. Can the contents of this file please move into `modules/helpers.js`?